submariner/multiple-cable-drivers.md

```diff
@@ -0,0 +1,415 @@
+# Support fine-grained cluster connectivity and multiple cable drivers
+
+## Summary
+
+Submariner currently assumes full mesh connectivity by default and provisions VPN connections between all participating
+clusters. In addition, Submariner currently offers one of three cable drivers: VXLAN, IPSec or wireguard with limited
+configuration options. Only one of these cable drivers is enabled across all the clusters in a cluster set, that is all
+clusters that join a broker must use the same cable driver. The goal of this proposal is to outline an extended model
+that will support:
+
+- The selection of which clusters to connect and in what fashion (full mesh or connection to subset of clusters in
```

Comment:
Why is this necessary?

A clusterset is, by our definition, "one reachable space". If we're starting to segment that, then why even use submariner?

I'm talking about L3 connectivity, not ACLs which could block or allow certain types of communication.

Reply:
Perhaps my understanding of the MCS definition of a ClusterSet is in conflict with the Submariner definition. Where is the Submariner definition described?

Reply:
You can see it here https://submariner.io/getting-started/architecture/